app/src/main/res/values-en/strings.xml

```diff
@@ -902,9 +902,9 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_fireHydrant_type_wall">Wall</string>
     <string name="quest_fireHydrant_type_pond">Pond</string>
 
-    <string name="quest_footwayPartSurface_title">What’s the surface of the footway here?</string>
+    <string name="quest_footwayPartSurface_title">What’s the surface of the footpath here?</string>
 
-    <string name="quest_fuelSelfService_title">Can you pump gas yourself here?</string>
+    <string name="quest_fuelSelfService_title">Can you pump your own gas here?</string>
```

Comment:
Sorry @westnordost but I strongly disagree with your last point. en-us should be for US users to localise for their region, not for a generic global English. Generic global English should be simply "en" or "English (Global)".

Reply:
Either way already seems like a compromise away from the idiomatic American English: “Does this gas station have self-service?”

Reply:
Yep, US gas stations are "self-service", "full-service", or have both options.  Typically labeled "SELF-SERVE" in big friendly letters.  The way @1ec5 phrased it is the way I would use it in everyday life.

Reply:
> Generic global English should be simply "en" or "English (Global)".

Maybe it got improved but last time I checked there was no such thing in Android, with USA English treated as a global English.

Winning cultural imperialism wars also has negative effects :)

Reply:
Hm, I wonder if the suggestion made by @1ec5 is already clearer for US speakers, why not use this wording as this is exactly what is tagged too `self_service=*`.

Maybe it was reworded because of https://github.com/streetcomplete/StreetComplete/pull/2822#issuecomment-1008438719 

Here is another post where I asked on slack in #language about the wording: https://github.com/streetcomplete/StreetComplete/pull/2822#discussion_r786147298

Reply:
Self-service makes sense to me!